cernopendata/modules/fixtures/data/records/cms-validated-runs-Run2012B.json

```diff
@@ -0,0 +1,66 @@
+[
+{
+  "abstract": {
+    "description": "This file describes which luminosity sections in which runs are considered good and should be processed."
+  },
+  "accelerator": "CERN-LHC",
+  "collaboration": {
+    "name": "CMS Collaboration",
+    "recid": "451"
+  },
+  "collections": [
+    "CMS-Validated-Runs",
+    "CMS-Validation-Utilities"
+  ],
+  "date_created": "2012",
+  "date_published": "2017",
+  "distribution": {
+    "formats": [
+      "txt"
+    ]
+  },
+  "doi": "10.7483/OPENDATA.CMS.C00V.SE32",
+  "experiment": "CMS",
+  "files": [
+    {
+      "checksum": "sha1:b24efcfb1050b083aff2390df01fc16d6b67b836",
+      "size": 36630,
+      "uri": "root://eospublic.cern.ch//eos/opendata/cms/validated-runs/2012/Cert_190456-208686_8TeV_22Jan2013ReReco_Collisions12_JSON.txt"
+    }
+  ],
+  "note": {
+    "description": "The list covers all p-p data taking in 2012. The public data (RunB and RunC) are part of this list, between run numbers 193833 - 196531 for RunB and 198022 - 203742 for RunC."
+  },
+  "publisher": "CERN Open Data Portal",
+  "recid": "1002",
+  "run_period": "Run2012B-Run2012C",
+  "title": "CMS list of validated runs Cert_190456-208686_8TeV_22Jan2013ReReco_Collisions12_JSON.txt",
+  "title_additional": "CMS list of validated runs for primary datasets of 2012 data taking",
+  "type": {
+    "primary": "Software",
```

Comment:
Please change primary type to "Environment" as per https://github.com/cernopendata/opendata.cern.ch/issues/1846#issuecomment-346081338.

The secondary type can stay "Validation".